### ISSUE TITLE
elementor widget for (easy) open street map integration [suggestion]

### ISSUE THREAD

**Issue by dasigna** [pojome/elementor#6033]
## Prerequisites

- [x] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [x] The feature is still missing in the latest stable version of Elementor ( Elementor Pro. )

**What problem is your feature request going to solve? Please describe.**
for me it would be a great idea to expand the maps widget for open street maps - or even better introduce a new widget for it.
google maps get boring more and more with that silly API policy and is a constant issue.
theres no real good solution for open street maps so far, so i think this would be great idea anyway as its time for an alternative optio with the best page builder on wp :)

**Describe the solution you'd like**
new widget for easy use of open street maps on any website.

**Describe alternatives you've considered**
external plugins... mostly not very pleasant, at best bet average support and seldom easy to use.

**Additional context**
Theres a hole to fill ...

**Comment by davidefiorio** [comment]
And also, ability to add more than one pin on map :)

**Comment by drnickyoung** [comment]
I would love to see an alternative to the Elementor Google Map widget. I have tried many of the WordPress plugins that use OpenStreetMap/OpenLayers/Leaflet and none work well with Elementor.

**Comment by simonbouchard** [comment]
Since the Google Maps pricing modifications, that would be a must have!

**Comment by fhfournier** [comment]
@simonbouchard it's true. Elementor should have GoogleMap/OpenStreetMap option!

**Comment by mihdan** [comment]
[Elementor Yandex Maps Widget](https://wordpress.org/plugins/mihdan-elementor-yandex-maps/)

**Comment by grrog** [comment]
Yes OpenStreetMap needs to be added

**Comment by mo-ghorbani** [comment]
Of course it need to be add !!!

**Comment by albertvisuals** [comment]
Yes, we really need an Open Street Map integration!

**Comment by albertvisuals** [comment]
Absolutely! We need an OSM widget with the option to fill it dynamically

**Comment by michal-madrzyk** [comment]
This ticket is pretty old.. will it be addressed finally?

**Comment by AlbanVidal** [comment]
2021 is here ! :)
Can we have any information about this Issue please ? (planned/delayed/...)

**Comment by Proktologe** [comment]
It **can't be that hard** to integrate a **Open Street Map**! 
Its a very good alternativ to Google

**Comment by lala81996** [comment]
I agree. Is there any update?

**Comment by Ephesian** [comment]
There is a plugin called OSM Map Widget for Elementor, which I have used.
https://wordpress.org/plugins/osm-map-elementor/

I am also working on providing extra functionality for layers provided in GeoJson files.

**Comment by JonnyQuinn1977** [comment]
A Bit late to the game, but thought it might be valuable for anyone who stumbles across this.

This works well https://wordpress.org/plugins/themecanvas-locations-for-elementor/
